testlib/resources.go

```diff
@@ -34,11 +34,38 @@ type testResourceDetails struct {
 	action  int8
 }
 
+// commonBaseSearchPaths is a custom version of what fileutils exposes. At some point, consolidate.
+var commonBaseSearchPaths = []string{
+	".",
+	"..",
+	"../..",
+	"../../..",
+	"../../../..",
+}
+
+func findFile(path string) string {
+	return fileutils.FindPath(path, commonBaseSearchPaths, func(fileInfo os.FileInfo) bool {
+		return !fileInfo.IsDir()
+	})
+}
+
+func findDir(dir string) (string, bool) {
```

Comment:
Can you explain how these changes relate to the rest of the PR?

Reply:
Basically, I just needed to search "up" a few more directories than `utils.FindFile` does today. I'd rather modify /it/ to search to `/` (maybe with a high upper limit like 50 to prevent a weird filesystem issue), but that would technically change the semantics of how we locate configuration files, etc.